Assets/MixedRealityToolkit/Inspectors/ServiceInspectors/HandJointServiceInspector.cs

```diff
@@ -0,0 +1,256 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using System.Collections.Generic;
+using System;
+using UnityEditor;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    [MixedRealityServiceInspector(typeof(HandJointService))]
+    public class HandJointServiceInspector : BaseMixedRealityServiceInspector
+    {
+        private const string ShowHandPreviewInSceneViewKey = "MRTK_HandJointServiceInspector_ShowHandPreviewInSceneViewKey";
```

Comment:
Private fields should start with lower case. See [coding guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/CodingGuidelines.html#naming-conventions)

Reply:
This is following a convention set by other inspectors. No issue with making them lower case, but we should do it in another PR where we change the other inspectors to be consistent.